### DISCUSSION TITLE
It is possible for two pcs to generate duplicated data ?

### DISCUSSION THREAD

**Discussion by rlopezpirez** [Chia-Network/chia-blockchain#2974] [Q&A]
Hello, im a newbie in this cryptocurrency and i have a big question ... 
If  i have 2 or more machines adding plots simultaneously with the same seed, the 2 plots generated can cointain duplicated information ?

it is better to create the plots on a single pc ??

**Comment by dorkmo** [comment]
theyll be unique, but still reward to your same wallet
